modules/apps/headless/headless-user-notification/headless-user-notification-test/src/testFunctional/macros/UserNotificationAPI.macro

```diff
@@ -48,26 +48,48 @@ definition {
 	}
 
 	macro createNotificationTemplate {
-		Variables.assertDefined(parameterList = "${name},${recipientType},${userScreenName},${en_US},${type}");
-
-		var curl = UserNotificationAPI._curlNotificationTemplates();
-		var body = '''
-			-d {
-				"editorType": "richText",
-				"externalReferenceCode": "${externalReferenceCode}",
-				"name": "${name}",
-				"recipientType": "${recipientType}",
-				"recipients": [
-					{
-						"userScreenName": "${userScreenName}"
-					}
-				],
-				"subject": {
-					"en_US": "${en_US}"
-				},
-				"type": "${type}"
-			}
-		''';
+		Variables.assertDefined(parameterList = "${name},${recipientType},${en_US},${type}");
+
+		if (${recipientType} == "term") {
```

Comment:
For made this more intuitive can you change the conditional from "term" to "Definition of Terms".